.travis.yml

```diff
@@ -19,12 +19,20 @@ matrix:
   allow_failures:
     - php: 7.1
 
+before_install:
+  - composer global require hirak/prestissimo
+
 before_script:
   - composer install --prefer-dist --no-interaction
 
 script:
   - if [[ $DEFAULT == 1 ]]; then vendor/bin/codecept run ; fi
   - if [[ $PHPCS == 1 ]]; then composer cs-check ; fi
 
+cache:
+  directories:
+    - vendor
+    - $HOME/.composer/cache
```

Comment:
Vendor would be useful to store in the other repos, we will migrate that into everything.

If your project use mailcatcher or some ruby gems (even unintentionally) look at adding:

`        - /home/travis/.rvm/gems`

Reply:
Something worth investigating with regards to `vendor` caching: does travis cache key take the branch into account? if not, you'll find a lot of cache misses and extra downloads.